hawkeye-api/src/handlers.rs

```diff
@@ -363,50 +369,59 @@ pub async fn start_watcher(id: String, client: Client) -> Result<impl warp::Repl
             ))
         }
     };
+
+    // Actions and guards based on the current Watcher status.
     match deployment.get_watcher_status() {
         Status::Running => Ok(reply::with_status(
+            // No op, already running!
             reply::json(&json!({
                 "message": "Watcher is already running"
             })),
             StatusCode::OK,
         )),
         Status::Pending => Ok(reply::with_status(
+            // No op, committing other changes.
             reply::json(&json!({
-                "message": "Watcher is updating"
+                "message": "Watcher is currently updating"
             })),
             StatusCode::CONFLICT,
         )),
         Status::Ready => {
-            // Start watcher / replicas to 1
-            let patch_params = PatchParams::default();
-
-            let fs = json!({
-                "spec": { "replicas": 1 }
+            // Start Watcher by setting Kubernetes deployment replicas=1
+            let mut patch_params = PatchParams::default();
+            patch_params.field_manager = Option::from("hawkeye_api".to_string());
```

Comment:
Maybe this could be simplified to:
```suggestion
            patch_params.field_manager = Some("hawkeye_api".to_string());
```

Reply:
Another option is:
```suggestion
            patch_params.field_manager = "hawkeye_api".to_string().into();
```